### PR TITLE
User delete

### PR DESCRIPTION
**Pull request by JennaDalgety**
Created member delete functionality for superAdmins

### CODE REVIEW

File: src/components/profile.vue [hunk 1]
```diff
@@ -89,266 +90,288 @@
...
+        profile: 'active',
+        camp: 'inactive',
+        volunteer: 'inactive',
+        partner: 'inactive'
+      },
+      userStatus: ''
+    };
+  },
+  methods: {
+    // changeStatus: function(link) {
```
Comment (mkirlin) [inline src/components/profile.vue]: If this is unused, let's delete it.

File: src/components/profile.vue [hunk 2]
```diff
@@ -89,266 +90,288 @@
...
+  justify-content: center;
+  padding-bottom: 60px;
+}
+.boxes {
+  background-color: white;
+  border: 5px solid rgb(140, 218, 192);
+  border-radius: 12px;
+  display: inline-block;
+}
+/* #sideBar {
```
Comment (mkirlin) [inline src/components/profile.vue]: Let's delete this rather than just commenting it out.

File: src/components/profile.vue [hunk 3]
```diff
@@ -89,266 +90,288 @@
...
+}
+#titleDiv {
+  display: inline-block;
+  margin-left: 5%;
+}
+#editDiv {
+  padding-top: 15px;
+  margin-right: 5%;
+}
+/* .profileNav {
```
Comment (mkirlin) [inline src/components/profile.vue]: Same as above.

**Review by mkirlin — CHANGES_REQUESTED** [review]

**Comment by JennaDalgety** [comment]
Deleted all commented out code that duplicated the separate sidebar component

**Pull request closed without merging**